config/plugins/visualizations/h5web/config/h5web.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="H5Web" embeddable="false">
+    <description>HDF5 data visualization and exploration</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">binary.H5</test>
+        </data_source>
+    </data_sources>
+    <params>
+        <param type="dataset" var_name_in_template="hda" required="true">dataset_id</param>
+    </params>
+    <entry_point entry_point_type="chart" src="script.js" css="script.css"/>
+    <settings>
+        <input>
+            <name>explorer</name>
+            <value>false</value>
```

Comment:
If you want to, you could also specify the type e.g. `<type>boolean</type>`.

Reply:
@jarrah42 this looks like a good recommendation.